geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp

```diff
@@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;
 // internals are prepared. fw_helper framework will handle this.
 CppCacheLibrary* CppCacheLibrary::initLib( void )
 {
-  ACE::init();
```

Comment:
Hi,

Yes, actually, I have 3 makefiles and some instructions for how to use them, but I didn’t put them in this PR since, as I understood from Jacob – you prefer to use the build.xml that exist in the branch.. I tried using it, but it took  too long to stabilize, so me and my colleagues wrote new makefiles.
Also – I have version.h (that contains versioning information) and config.h (which is empty) that were both needed for compilation, and I added them locally. I think both of these are also created by the build.xml when using it, so I didn’t add them here.

The reason for removing  the ACE:init() – we use ACE in our application as well, and therefore had to change/synchronize the version, remove it from the client library, and rely on the ACE initialization performed by our application.


Thanks,
Gal


From: Vishal Rao [mailto:notifications@github.com]
Sent: Wednesday, July 13, 2016 14:20
To: apache/incubator-geode
Cc: Gal Palmery; Author
Subject: Re: [apache/incubator-geode] GEODE-88: code fixes for c++ client (#199)


In geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp<https://github.com/apache/incubator-geode/pull/199#discussion_r70607228>:

> @@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;

>  // internals are prepared. fw_helper framework will handle this.

>  CppCacheLibrary* CppCacheLibrary::initLib( void )

>  {

> -  ACE::init();

Why remove the init here? Is it moved somewhere else and are there some files in this change not listed in the PR/diff maybe?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/apache/incubator-geode/pull/199/files/d70c5f917f3050365b5d4f6f81d38f26c9542481#r70607228>, or mute the thread<https://github.com/notifications/unsubscribe/ARLWfe7H8IGkh3prl8j_LkAUw8TqUChwks5qVMnQgaJpZM4JLJVC>.

This message and the information contained herein is proprietary and confidential and subject to the Amdocs policy statement,
you may review at http://www.amdocs.com/email_disclaimer.asp